### DISCUSSION TITLE
unocss & sass files

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [unocss/unocss#930] [Q&A] [answered]
I am using windicss and I have a bunch of sass files that are using @apply directive. Is it possible to replace unocss with windicss in this case?

**Comment by antfu** [comment, marked as answer]
Maybe you can use https://github.com/unocss/unocss/tree/main/packages/transformer-directives#css-variable-style so Sass will think they are valid CSS. And after the Sass compiler transforms it to CSS, then our transformer could expend those directives.
